### PR TITLE
changelog

### LINKED ISSUE #885: Set up a Change Log for abap2xlsx
Proposing to create a [`changelog.txt`](https://docs.abapgit.org/development/guide-changelog.html) file to display the latest changes of abap2xlsx when pulling the repository with abapGit.

Request from:
- @larshp (https://github.com/sapmentors/abap2xlsx/issues/720#issuecomment-757140976)
  > perhaps its time to keep some kind of changelog, https://docs.abapgit.org/development/guide-changelog.html
- @AndreaBorgia-Abo (https://github.com/sapmentors/abap2xlsx/pull/869#issuecomment-962605526)
  > this might be the right time to start a BREAKINGCHANGES file or something to that effect

@AndreaBorgia-Abo @larshp @gregorwolf Any idea which version number should be first used? 

One solution to get the first number is to take the [official release of abap2xlsx in July 12, 2010](https://blogs.sap.com/2010/07/12/abap2xlsx-generate-your-professional-excel-spreadsheet-from-abap/) as being version 1.0.0, we could count the number of months, Nov 12, 2021 would be 136 months after 1.0. So, version **1.136.0**?

After that we could release a new version at each beginning of month, as far as possible...

### PR DESCRIPTION
**Pull request by sandraros**
Proposing version `7.14.0`
Fix #885

### CODE REVIEW

File: src/zcl_excel_apack.clas.abap [hunk 1]
```diff
@@ -0,0 +1,32 @@
+CLASS zcl_excel_apack DEFINITION
+  PUBLIC
+  FINAL
+  CREATE PUBLIC .
+
+  PUBLIC SECTION.
+
+    INTERFACES if_apack_manifest.
```
Comment (larshp) [inline src/zcl_excel_apack.clas.abap:8]: I dont have the `if_apack_manifest` interface in my dev system

Comment (sandraros) [reply src/zcl_excel_apack.clas.abap:8]: oh, I remember that automatic renaming when the code is staged/pushed via abapGit, from `zif_apack_manifest` to `if_apack_manifest`, and when the code is pulled via abapGit, the opposite renaming is done from `if_apack_manifest` to `zif_apack_manifest`.

Comment (larshp) [reply src/zcl_excel_apack.clas.abap:8]: heh, yea, it works in mysterious ways 🤪

File: changelog.txt [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
++ Possibility to get the fieldcatalog with mandt
+! cleanup types in zif_excel_book_protection
+
+2021-01-11 v7.4.0
+------------------
+! warning about SAPLink in documentation, and replace obsolete links in zangry_birds and zdemo_excel1
+! remove XFELD usage
+! Dynamic prefill of date in zdemo_calendar
+
+2021-12-11 v7.3.0
```
Comment (AndreaBorgia-Abo) [inline changelog.txt:113]: Whoopsie, I noticed that the year is wrong here...

File: changelog.txt [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+------------------
+* Fix in ZDEMO_EXCEL11: Field-symbol usage
+! Update abapGit-installation.md
+
+2021-11-11 v7.2.0
```
Comment (AndreaBorgia-Abo) [inline changelog.txt:118]: ... and here

**Pull request merged**